examples/cassandra/README.md

```diff
@@ -364,9 +452,18 @@ Create this daemonset:
 $ kubectl create -f examples/cassandra/cassandra-daemonset.yaml
 ```
 
-Now if you list the pods in your cluster, and filter to the label ```name=cassandra```, you should see one cassandra pod for each node in your network:
+Make sure that your Kubernetes cluster has enabled use of the experimental API.
```

Comment:
Yes, this shouldn't be true at HEAD and in 1.2. Please try this on HEAD to confirm that it works, and remove this.